src/libraries/System.Private.CoreLib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -51,8 +51,14 @@ public Dictionary(int capacity, IEqualityComparer<TKey>? comparer)
 
             if (capacity > 0)
             {
-                Initialize(capacity);
+                Initialize(HashHelpers.GetPrime(capacity));
```

Comment:
Why has `HashHelpers.GetPrime` been moved out of the `Initialize` method? It doesn't like all callsites have been updated to reflect the change.

Reply:
It was moved out because `TrimExcess` was already passing in a size computed from `HashHelpers.GetPrime` so it was doing double work. That is the only callsite I did not update.